azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/toolkit/lib/auth/util/AzureEnvironmentUtils.java

```diff
@@ -52,6 +52,17 @@ public static void setupAzureEnvironment(AzureEnvironment env) {
             // src/main/java/com/azure/identity/implementation/IdentityClientOptions.java#L42
             Configuration.getGlobalConfiguration().put(Configuration.PROPERTY_AZURE_AUTHORITY_HOST, env.getActiveDirectoryEndpoint());
         }
+
+        final com.microsoft.azure.toolkit.lib.AzureConfiguration az = com.microsoft.azure.toolkit.lib.Azure.az().config();
```

Comment:
Please make sure the method is invoked in each tooling

Reply:
Auth lib will call this method before every login